final-project/styles/main.scss

```diff
@@ -1,13 +1,745 @@
+  list-style-type: none;
+  width: 70%;
+  margin: 5% 0;
+  &__item{
+    &__question{
+      display: flex;
+      margin: 10px 0;
+      text-align: left;
+      i{
+        font-size: 1.5rem;
+      }
+      p{
+        margin-left: 10px;
+      }    
+    }
+    &__answer{
+      display: flex;
+      text-align: left;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      p{
+        width: 400px;
+        margin: 10px 25px;
+        font-size: .8rem;
+      }
+      button{
+        margin: 10px 20px;
+      }
+    }
+  }
+}
+
+.newsletter{
+  height: 350px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  &__content{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-around;
+    width: 70%;
+    &__description{
+      flex: auto;
+      width: 330px;
+      margin: 10px 5px;
+    }
+    &__form{
+      display: flex;
+      flex-wrap: wrap;
+      flex: auto;
+      input{
+        flex: auto;
+        width: 280px;
+        margin: 5px;
+      }
+      button{
+        flex: auto;
+        margin: 5px;
+      }
+    }
+  }
+}
+
+.footer{
+  width: 100%;
+  font-family: Quicksand,sans-serif;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  &__logo{
+    width: 26px;
+    min-width: 26px;
+    min-height: 26px;
+    background-color: #5e81fe;
+    border-radius: 50%;
+  }
+  &__content{
+    margin: 30px 0;
+    width: 80%;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+  }
+  &__section{
+    width: 300px;
+    display: flex;
+    flex: auto;
+  }
+  &__nav-list{
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    &__item{
+      flex:auto;
+      text-decoration: none;
+      color: #697CA6;
+      cursor: pointer;
+      &__selected, &:hover{
+        text-decoration: none;
+        color: #5E81FE;
+      }
+    }
+  }
+}
+.copyright{
+  margin: 20px 0;
+  p{
+    text-align: center;
+  }
+}
+
+// IMAGES
+#bit-bucket{
+  background-image: url("/reference/assets/customers/bitbucket.png");
+}
+#apple-watch{
+  background-image: url("/reference/assets/customers/apple-watch.png");
+}
+#facebook{
+  background-image: url("/reference/assets/customers/facebook.png");
+}
+#atlassian{
+  background-image: url("/reference/assets/customers/atlassian.png");
+}
+#audi{
+  background-image: url("/reference/assets/customers/audi.png");
+}
+#home-icon{
+  background-image: url("/reference/assets/icons/home.png");
+}
+#folder-icon{
+  background-image: url("/reference/assets/icons/folder.png");
+}
+#chat-icon{
+  background-image: url("/reference/assets/icons/chat.png");
+}
+#upload-icon{
+  background-image: url("/reference/assets/icons/upload.png");
+}
+#blue-clouds{
+  background-image: url("/reference/assets/blue-clouds-bg.png");
+}
+
+// MEDIA QUERIES
+
+/* Tablet Landscape */
+@media only screen 
```

Comment:
For readability I would recommend to have these media query rules near their base rules.
Also scss files can be imported so you can separate your rules by section instead of just one big file